src/py42/services/legalhold.py

```diff
@@ -215,6 +219,10 @@ def get_custodians_page(
         Returns:
             :class:`py42.response.Py42Response`:
         """
+        if not any((legal_hold_membership_uid, legal_hold_uid, user_uid, user)):
```

Comment:
Is there an HTTP error that occurs that is specific when none of these are received?

Reply:
I ask only because it might nice to let it occur and use it as the base class so it works similarly to before, but if that is not the case, this is probably better

Reply:
The error response is `Py42BadRequestError` and does have a specific error message:

```
'[{"name":"SYSTEM","description":"Invalid request: SYSTEM com.code42.core.BuilderException: At least one criteria must be specified; holdMembershipUid, holdUid, userUid, or userSearch"}]'
```

I reworked it to be a custom exception inheriting from Py42BadRequestError so anyone depending on that behavior won't run into problems.